metaworld/envs/mujoco/sawyer_xyz/sawyer_assembly_peg.py

```diff
@@ -43,19 +44,21 @@ def __init__(self, random_init=True):
             np.array([1, 1, 1, 1]),
         )
 
-        goal_low = np.array(goal_low)
-        goal_high = np.array(goal_high)
+        self.goal_low = np.array(self.goal_low)
+        self.goal_high = np.array(self.goal_high)
         self.obj_and_goal_space = Box(
-            np.hstack((obj_low, goal_low)),
-            np.hstack((obj_high, goal_high)),
+            np.hstack((obj_low, self.goal_low)),
+            np.hstack((obj_high, self.goal_high)),
         )
-        self.goal_space = Box(np.array(goal_low), np.array(goal_high))
 
         self.observation_space = Box(
             np.hstack((self.hand_low, obj_low,)),
             np.hstack((self.hand_high, obj_high,)),
         )
+
+        self._freeze_rand_vec = False
```

Comment:
Y'know, we probably could have just put this bit of logic in the base class `reset`. We can fix that in a later PR.